### DISCUSSION TITLE
Laporan dan Kehadiran

### DISCUSSION THREAD

**Discussion by muhdavi** [muhdavi/laporan-practice-big-data#34] [Show and tell]
Mulai tanggal 1 Desember 2022 presensi dan laporan akan dibahas disini. Setiap selesai pertemuan Dosen Pengampu akan mempublish mahasiswa-mahasiswa yang hadir. Selanjutnya sebelum pertemuan minggu berikutnya dimulai, mahasiswa sudah mengirimkan laporan.

Format _message_ presensi ketika _commit_:
_"Presensi dd-mm-yyyy a.n. Nama"_

Format _message_ laporan ketika _commit_:
_"Laporan dd-mm-yyyy a.n. Nama"_

**Comment by muhdavi** [comment]
Kehadiran tanggal 1-12-31

1. @AdindaAwaliah 
2. @AryaSpt -> Izin
3. @cutopymdl 
4. @jhndsrh 
5. @Muhammadikhsandev 
6. @Ikram160302
7. @Munawir027 
8. @NadzuraKumaira 
9. @fardaniahnh 
10. @rauzatinursyah 
11. @resharussita 
12. @RIZKIINC 
13. @salsabilairmanda17 
14. @SyarfaniAkbar
15. @traviafzah 
16. @zulfahmidev

**Comment by muhdavi** [comment]
Kehadiran tanggal 2-12-31

1. @Munawir027 
2. @RIZKIINC 
3. @rauzatinursyah 
4. @salsabilairmanda17 
5. @AdjieYusmunandar 
6. @fardaniahnh 
7. @cutopymdl 
8. @resharussita 
9. @traviafzah 
10. @AdindaAwaliah 
11. @jhndsrh 
12. @Muhammadikhsandev 
13. @zulfahmidev

**Comment by muhdavi** [comment]
Kehadiran tanggal 8-12-2022

1. @AdindaAwaliah 
2. @AdjieYusmunandar 
3. @cutopymdl 
4. @jhndsrh 
5. @Muhammadikhsandev 
6. @Ikram160302
7. @Munawir027 
8. @fardaniahnh 
9. @rauzatinursyah 
10. @resharussita 
11. @RIZKIINC 
12. @salsabilairmanda17 
13. @traviafzah 
14. @zulfahmidev

**Comment by muhdavi** [comment]
Kehadiran tanggal 9-12-2022

1. @AdindaAwaliah
2. @AdjieYusmunandar 
3. @cutopymdl 
4. @jhndsrh 
5. @Muhammadikhsandev 
6. @Munawir027 
7. @fardaniahnh 
8. @rauzatinursyah 
9. @resharussita 
10. @RIZKIINC 
11. @salsabilairmanda17 
12. @traviafzah 
13. @zulfahmidev

**Comment by muhdavi** [comment]
Kehadiran tanggal 15-12-2022

1.  @AdindaAwaliah 
2. @AdjieYusmunandar 
3. @AryaSpt 
4. @cutopymdl 
5. @faizayuwafiqi 
6. @jhndsrh 
7. @Muhammadikhsandev 
8. @Ikram160302
9. @Munawir027 
10. @NadzuraKumaira 
11. @fardaniahnh 
12. @Nurulaflahhh
13. @rauzatinursyah 
14. @resharussita 
15. @RIZKIINC 
16. @salsabilairmanda17 
17. @traviafzah

**Comment by muhdavi** [comment]
Kehadiran tanggal 16-12-2022

1. @AdindaAwaliah 
2. @AdjieYusmunandar 
3. @faizayuwafiqi 
4. @jhndsrh 
5. @Muhammadikhsandev 
6. @fardaniahnh 
7. @olala17 
8. @Nurulaflahhh
9. @rauzatinursyah 
10. @resharussita 
11. @RIZKIINC 
12. @salsabilairmanda17 
13. @traviafzah 
14. @zulfahmidev

**Reply by NadzuraKumaira** [reply]
15.@nadzurakumaira

**Reply by muhdavi** [reply]
Saya cek di Pull Request kok ga ada ya @NadzuraKumaira ? Kalau memang ada tolong infokan ke saya ya.
